### DISCUSSION TITLE
Integration für HomeAssistant

### DISCUSSION THREAD

**Discussion by joschua8** [marudor/bahn.expert#715] [Ideas]
Erstmal vielen Dank für die sehr gute Webseite.

Ich weiß nicht wie realistisch es ist, aber ist eine HA Integration möglich?

Die in HA vorhandene Deutsche Bahn Integration wurde/wird eingestellt...

**Comment by moorjan** [comment]
Hi, ich nutze db-infoscreen, damit bin ich eigentlich zufrieden. Dort fehlen mir zwar vereinzelt Funktionionen aber es geht.
